src/components/EthExchanges.tsx

```diff
@@ -543,9 +558,15 @@ const EthExchanges = () => {
   const exchangesArray = Object.keys(exchanges)
   const walletProvidersArray = Object.keys(walletProviders)
   // Construct arrays for CardList
-  let filteredExchanges = []
-  let filteredWalletProviders = []
-  let filteredWallets = []
+  let filteredExchanges: Array<{
```

Comment:
I think you can get away with `Array<Wallet>` here

Reply:
Yeah that's what I thought but I figured it could be confusing.
I'm gonna fix it!